LICENSE.txt

```diff
@@ -0,0 +1,21 @@
+
+            
+            Copyright (c) 2017 anisabboud
+            
+            Permission is hereby granted, free of charge, to any person obtaining a copy
+            of this software and associated documentation existss (the "Software"), to deal
```

Comment:
The word "existss" should be "files" according to https://opensource.org/licenses/MIT.